packages/core/pubspec.yaml

```diff
@@ -24,7 +24,7 @@ dependencies:
   json_annotation: ^4.4.0
   glob: ^2.0.0
 
-  dart_style: '>=2.2.4 <4.0.0'
+  dart_style: '>=2.2.4 <3.0.0'
```

Comment:
Sorry. I had overlooked that `languageVersion` is optional in v2.3.7 but requierd in v3.0.0.

https://github.com/dart-lang/dart_style/blob/v2.3.7/lib/src/dart_formatter.dart#L83
https://github.com/dart-lang/dart_style/blob/v3.0.0/lib/src/dart_formatter.dart#L82